testData/highlighting/equalinif.go

```diff
@@ -0,0 +1,29 @@
+package equalinif
+
+func <warning descr="Unused function 'eqlfunc'">eqlfunc</warning>() {
```

Comment:
Can you please give me an example? I'm not sure I understand what do you mean. Thank you.